packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/types/purchase_details.dart

```diff
@@ -37,7 +38,7 @@ class PurchaseDetails {
   final String? transactionDate;
 
   /// The status that this [PurchaseDetails] is currently on.
-  PurchaseStatus? status;
+  PurchaseStatus status;
```

Comment:
this is added in a separate PR.

Reply:
Correct, I still needed to merge with master. This should be gone now.